### PR TITLE
datepicker

### PR DESCRIPTION
**Pull request by VortexExpansion**
#18 

@yabe-diverta san 
https://diverta.gyazo.com/744da4236120def49005c9468d7073f3

### CODE REVIEW

**Review by yabe-diverta — CHANGES_REQUESTED** [review]
Could you add i18n plugin for Japanese?

**Comment by VortexExpansion** [comment]
But front_nuxt_corporate is already fully in japanese.

**Comment by yabe-diverta** [comment]
@VortexExpansion 
I thought the datepicker should be Japanese, not the other elements.
Am I missing something?

**Comment by VortexExpansion** [comment]
Oh maybe I see what you mean. 
This site (unlike front_nuxt_auth) has no language button for switching. 

But maybe you mean keep the datepicker only in japanese and not in default english implementation right ?

**Comment by yabe-diverta** [comment]
@VortexExpansion 
exactly~~~

**Comment by VortexExpansion** [comment]
@yabe-diverta san 
Added https://diverta.gyazo.com/2cd2f687d93333665a9d42114c31d7a0
Please confirm.

File: pages/contact/index.vue
Comment (yabe-diverta) [inline pages/contact/index.vue:1]: When it comes to typical custom about Japanese calendar,
- weeks start with Sunday
- ordering for date string differs to West (yyyy/MM/dd is JP, MM/dd/yyyy is EN)
https://diverta.gyazo.com/df8e12dfcca875f3b1739cc5696666c9

**Review by yabe-diverta — CHANGES_REQUESTED** [review]

**Comment by VortexExpansion** [comment]
@yabe-diverta  san 

Formatted according to typical Japanese calendar. https://diverta.gyazo.com/11687d4253544c10dff68c60d97a26bd
Please check now

**Review by yabe-diverta — APPROVED** [review]
Thank you to fix, no issues 🙇

**Comment by VortexExpansion** [comment]
@HisashiHasebe san 
Please confirm and merge

**Pull request merged**